packages/auth/src/grant/routes.ts

```diff
@@ -101,7 +102,12 @@ async function createGrant(
   try {
     noInteractionRequired = canSkipInteraction(deps.config, ctx.request.body)
   } catch (err) {
-    ctx.throw(400, 'identifier_required', { error: 'identifier_required' })
```

Comment:
I have no idea where I got stuff like `identifier_required` or `internal_server_error`. I've restricted error responses to the ones that are listed in https://datatracker.ietf.org/doc/html/draft-ietf-gnap-core-protocol-18#section-3.6